samples/detect.v1p1beta1.js

```diff
@@ -73,240 +69,9 @@ function detectText(fileName) {
   // [END vision_detect_document]
 }
 
-
-function detectFaces(fileName) {
```

Comment:
Is the rationale for the removal of these samples that you are _only_ showing samples added or changed in 1.1?

Reply:
Correct, these are only Beta samples. Samples in the repo exist to support documentation. The Beta documentation displays 4 enhancements as well as changed import statements to get the Beta library explicitly